server/CGameHandler.cpp

```diff
@@ -1195,11 +1195,11 @@ void CGameHandler::handleClientDisconnection(std::shared_ptr<CConnection> c)
 {
 	for(auto playerConns : connections)
 	{
-		for(auto conn : playerConns.second)
+		for(auto i = playerConns.second.begin(); i != playerConns.second.end(); )
```

Comment:
All these ".first" and ".second" to me are hard to look at.  Would be great to refactor this code giving human readable names.